GSA-TTS/FAC
# Sk/csv str,error table mod

https://github.com/GSA-TTS/FAC/issues/2993
https://github.com/GSA-TTS/FAC/issues/2977

Test:
1. 2993 -

- Load data into db using procedure https://github.com/GSA-TTS/FAC/blob/main/backend/census_historical_migration/README.md#how-to-load-test-census-data-into-postgres
- Run the following query to verify if leading zeroes got retained for zipcode - 
select length(chme."ZIPCODE"), count(*)  from census_historical_migration_elecauditheader chme 
group by length(chme."ZIPCODE")
- Verify that the query result is as follows -
<img width="271" alt="Screenshot 2023-12-14 at 9 18 56 AM" src="https://github.com/GSA-TTS/FAC/assets/135276194/ddb9044e-6c05-44ec-aeb6-052cc69f104a">

2. 2977 - 

- Verify that the MigrationErrorDetail table has the following columns:
<img width="415" alt="Screenshot 2023-12-14 at 9 22 15 AM" src="https://github.com/GSA-TTS/FAC/assets/135276194/54a116d0-dd79-4a1e-9b2b-82e5c2fcb658">


## PR checklist: submitters







## PR checklist: reviewers


The larger the PR, the stricter we should be about these points.
